src/main/java/Baron.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Scanner;
```

Comment:
nice that you imported only java.util.Scanner instead of importing the full java.util library!